cmd/replication/main.go

```diff
@@ -49,17 +61,34 @@ func main() {
 		)
 
 		if err != nil {
-			log.Fatal("initializing database", zap.Error(err))
+			logger.Fatal("initializing database", zap.Error(err))
 		}
 
 		privateKey, err := utils.ParseEcdsaPrivateKey(options.SignerPrivateKey)
 		if err != nil {
 			log.Fatal("parsing private key", zap.Error(err))
 		}
 
+		var mtcs *metrics.Server
```

Comment:
I don't have strong opinions on this, but it would be fine to create the metrics server inside `pkg/server`. It might make more logical sense given that the server is responsible for shutting down the metrics.